From e00e170936e03d168d45451b525d9eef20d9afb1 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Tue, 19 Aug 2014 22:03:05 -0400
Subject: [redhat] Kconfig: Enable CONFIG_SIGNED_PE_FILE_VERIFICATION=y

Message-id: <1408485785-777-34-git-send-email-vgoyal@redhat.com>
Patchwork-id: 88392
O-Subject: [RHEL7.1 PATCH 33/33] config: Enable CONFIG_SIGNED_PE_FILE_VERIFICATION=y
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ:1129779

Kdump patches later will require that PE signed bzImage signature can
be verified and we will need CONFIG_SIGNED_PE_FILE_VERIFICATION=y.

Signed-off-by: Vivek Goyal <vgoyal@redhat.com>
---
 create mode 100644 redhat/configs/generic/x86_64/CONFIG_SIGNED_PE_FILE_VERIFICATION

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_SIGNED_PE_FILE_VERIFICATION b/redhat/configs/generic/x86_64/CONFIG_SIGNED_PE_FILE_VERIFICATION
new file mode 100644
index 0000000..d7551a5
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_SIGNED_PE_FILE_VERIFICATION
@@ -0,0 +1 @@
+CONFIG_SIGNED_PE_FILE_VERIFICATION=y
-- 
1.7.1